src/environment/openfin-env.ts

```diff
@@ -50,6 +50,11 @@ export default class OpenFinEnvironment implements Environment {
          });
     }
 
+    public getRandomId = (): string => {
+        const intArray = new Uint32Array(1);
+        return window.crypto.getRandomValues(intArray)[0].toString(32);
```

Comment:
did you build it? I get the following error on this line:

src/environment/openfin-env.ts(55,16): error TS7017: Element implicitly has an 'any' type because type 'Int8Array | Uint8Array | Uint8ClampedArray | Int16Array | Uint16Array | Int32Array | Uint32Array .
..' has no index signature

Reply:
Don't see build task or editor TS errors on my side... However, I added a type now so hope it helps. Can't verify if it fixes because I don't get the error any way.

![screen shot 2018-03-28 at 9 44 42 am](https://user-images.githubusercontent.com/5790216/38033155-5da81f24-326d-11e8-8937-23075137958f.png)

Reply:
Also Travis CI for that commit shows a successful build